entry_tasks/3_sum_numbers/3_sum_numbers.py

```diff
@@ -0,0 +1,13 @@
+if __name__ == '__main__':
+    #Create python program which sums 2 digits and prints result to console
```

Comment:
Please, remove this comments, you have a reference in the commit to an issue, no need to duplicate the details